tests/test_md_cli.py

```diff
@@ -342,3 +342,31 @@ def test_invalid_config():
     )
     assert result.exit_code == 1
     assert isinstance(result.exception, ValueError)
+
+
+def test_struct_name(tmp_path):
+    """ "Test specifying the structure name."""
+    stats_path = tmp_path / "EXAMPLE-nvt-T10.0-stats.dat"
+    traj_path = tmp_path / "EXAMPLE-nvt-T10.0-traj.xyz"
+    final_path = tmp_path / "EXAMPLE-nvt-T10.0-final.xyz"
+    struct_name = tmp_path / "EXAMPLE"
```

Comment:
While they're constant, I would prefer to see the other paths derived from `struct_name`
```suggestion
    struct_name = "EXAMPLE"
    struct_path = tmp_path / struct_name
    stats_path = tmp_path / f"{struct_name}-nvt-T10.0-stats.dat"
    traj_path = tmp_path / f"{struct_name}-nvt-T10.0-traj.xyz"
    final_path = tmp_path / f"{struct_name}-nvt-T10.0-final.xyz"
```

The reason being that it better demonstrates the meaning of the variable and thus serves as partial documentation.

Reply:
Good idea, thanks